jena-arq/src/main/java/org/apache/jena/riot/out/NodeFormatterTTL_MultiLine.java

```diff
@@ -18,95 +18,120 @@
 
 package org.apache.jena.riot.out;
 
-import static org.apache.jena.atlas.lib.Chars.CH_QUOTE1 ;
-import static org.apache.jena.atlas.lib.Chars.CH_QUOTE2 ;
-
-import org.apache.jena.atlas.io.AWriter ;
-import org.apache.jena.atlas.io.IndentedWriter ;
-import org.apache.jena.riot.out.quoted.QuotedStringOutput ;
-import org.apache.jena.riot.out.quoted.QuotedStringOutputTTL_MultiLine ;
-import org.apache.jena.riot.system.PrefixMap ;
-
-/** 
- * Node formatter for pretty-printed Turtle.
- *  This {@link NodeFormatter} switches between " and ' quotes to avoid escapes.
- *  This code writes multiline literals with """ or '''. 
+import static org.apache.jena.atlas.lib.Chars.CH_QUOTE1;
+import static org.apache.jena.atlas.lib.Chars.CH_QUOTE2;
+
+import org.apache.jena.atlas.io.AWriter;
+import org.apache.jena.atlas.io.IndentedWriter;
+import org.apache.jena.riot.out.quoted.QuotedStringOutput;
+import org.apache.jena.riot.out.quoted.QuotedStringOutputTTL_MultiLine;
+import org.apache.jena.riot.system.PrefixMap;
+
+/**
+ * Node formatter for pretty-printed Turtle. This {@link NodeFormatter} switches
+ * between " and ' quotes to avoid escapes. This code writes multiline literals with
+ * """ or '''.
  */
 public class NodeFormatterTTL_MultiLine extends NodeFormatterTTL {
-    /// For '-quoted and "-quoted literals 
-    private final QuotedStringOutput escapeProc2 = new QuotedStringOutputTTL_MultiLine(CH_QUOTE2) ;
-    private final QuotedStringOutput escapeProc1 = new QuotedStringOutputTTL_MultiLine(CH_QUOTE1) ;
+    /// For '-quoted and "-quoted literals
+    private final QuotedStringOutput escapeProc2 = new QuotedStringOutputTTL_MultiLine(CH_QUOTE2);
+    private final QuotedStringOutput escapeProc1 = new QuotedStringOutputTTL_MultiLine(CH_QUOTE1);
 
     private boolean writeAsMultiLine(AWriter w, String lex) {
-        return ( lex.contains("\n") ) ;
+        return (lex.contains("\n"));
     }
 
     public NodeFormatterTTL_MultiLine(String baseIRI, PrefixMap prefixMap) {
-        super(baseIRI, prefixMap) ;
+        super(baseIRI, prefixMap);
     }
 
     public NodeFormatterTTL_MultiLine(String baseIRI, PrefixMap prefixMap, NodeToLabel nodeToLabel) {
         super(baseIRI, prefixMap, nodeToLabel);
     }
 
-    private Runnable noop = ()->{} ;
+    private Runnable noop = () -> {};
```

Comment:
This `Runnable` can be declared `private static final`.

Reply:
Can do - the compiler will see it's not re-assigned and a constant anyway.